docs/localization.md

```diff
@@ -87,33 +85,40 @@ const locales = ['en', 'fr'] as const;
 The following example shows how messages can be used in **server** component:
 
 ```tsx
-import { NextIntlClientProvider } from 'next-intl';
-import { getMessages, getTranslations, unstable_setRequestLocale } from 'next-intl/server';
+import { getTranslations } from 'next-intl/server';
 // ...
-import { LocaleType } from '~/i18n';
 
-interface Props {
-  params: {
-    locale: LocaleType;
-  };
+export default async function Home() {
+  const t = await getTranslations('Home');
+  // ...
+
+  return (
+    <div>
+      <ProductCardCarousel
+        products={featuredProducts}
+        title={t('Carousel.featuredProducts')}
+      />
+    </div>
+  );
 }
+```
+
+**Client** and **shared** components use regular hooks to translate messages:
 
-export default async function Home({ params: { locale } }: Props) {
-  unstable_setRequestLocale(locale);
+```tsx
+'use client';
+
+import { useTranslations } from 'next-intl';
+// ...
 
-  const t = await getTranslations({ locale, namespace: 'Home' });
-  const messages = await getMessages({ locale });
+const Component = () => {
+  const t = await getTranslations('Cart');
```

Comment:
```suggestion
  const t = await useTranslations('Cart');
```

Reply:
😆 I changed the import but not the translation.